RTN-004.tex

```diff
@@ -53,7 +57,320 @@
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
 
- 
+\section{Introduction to Data Preview 0}\label{sec:intro}
+
+There are three planned Data Previews (DPs) during the period leading up to the start of Rubin Operations, termed pre-operations.
+DP0 will be based on simulated data.
```

Comment:
I realise this decision was taken at the Ops exec level but I want to express the disappointment of the technical teams. We believe that real sky precursor data (such as HSC)  would have resulted in a much more realistic rehearsal of the platform and also created interest amongs user groups that are unexcited by the simulated DC2 dataset. The risk here is that the support impact on the Science Platform and Science Pipelines teams exceeds the benefit of the feedback we get. I think it would be good to at least have an explanation here as to why only simulated data is being made available.

Reply:
An explanation of why simulated data is being used for DP0 is a paragraph I can't write on my own -- so am bringing this comment to the attention of @leannep and @rblum5.